build.yaml

```diff
@@ -586,6 +599,95 @@ common_cli:
     parse_logs vagrant box add --force "clusterhq/flocker-tutorial" \
       vagrant/tutorial/flocker-tutorial-\${FLOCKER_VERSION}.box
 
+  do_not_abort_on_errors: &do_not_abort_on_errors |
+    # make sure we don't abort the job on the first error we find
+    set +e
+
+  new_vagrantfile_for_yosemite: &new_vagrantfile_for_yosemite |
+    cat <<'EOF' >Vagrantfile
+    Vagrant.configure(2) do |config|
+       config.vm.box = 'jenkins-osx-yosemite'
+       config.vm.synced_folder '.', '/vagrant', disabled: true
+       config.vm.provider 'virtualbox' do |vb|
+          vb.memory = '1024'
+       end
+    end
+    EOF
+
```

Comment:
builds a new vagrantfile for yosemite using our overnight jenkins-osx-yosemite build